pontus/week2/Ass3/app.js

```diff
@@ -0,0 +1,241 @@
+class Wrapper extends React.Component { 
+	constructor(props) { 
+		super(props);
+
+      this.state = { lists:{} };
+
+		this.createList = () => { 
+			let newID = Date.now();
+			let lists = JSON.parse(JSON.stringify(this.state.lists));
+
+			if (this.refs.inputField.value != '') { 
+				let nameOfList = this.refs.inputField.value;
+				lists[newID] = { title: nameOfList };
+			}
+			else { 
+				lists[newID] = {};
+			}
+			this.setState({ lists });
+			this.refs.inputField.value = "";
+		}
+	}
+
+	moveMsg(currentListID, newListID, msgID) { 
+		let lists = JSON.parse(JSON.stringify(this.state.lists));
+
+		lists[newListID][msgID] = lists[currentListID][msgID];
```

Comment:
by building your data-model like this, it would be very hard/unreadable to do filters and in general get data out. You would have to iterate by keys/properties and that always(!) create more noise than necessary.
More complex code is always never better code in terms of readability and maintainability.